modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -214,6 +222,57 @@ public CompletableFuture<Void> dropColumn(AlterTableDropColumnParams params) {
         return failedFuture(new UnsupportedOperationException("Not implemented yet."));
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public CompletableFuture<Void> alterColumn(AlterColumnParams params) {
+        return saveUpdate(catalog -> {
+            String schemaName = Objects.requireNonNullElse(params.schemaName(), CatalogService.PUBLIC);
+
+            SchemaDescriptor schema = Objects.requireNonNull(catalog.schema(schemaName), "No schema found: " + schemaName);
+
+            TableDescriptor table = schema.table(params.tableName());
+
+            if (table == null) {
+                if (params.ifTableExists()) {
+                    return Collections.emptyList();
+                }
```

Comment:
"CREATE TABLE IF EXISTS" always throws an exception in such case.
@korlov42 , do you agree to pass 'if exists' flags inside the parameters?

Reply:
This check has been removed.